LeaderboardBackend.Test/Leaderboards.cs

```diff
@@ -62,25 +73,39 @@ public static async Task CreateLeaderboards_GetLeaderboards()
 		HashSet<Leaderboard> createdLeaderboards = new();
 		for (int i = 0; i < 5; i++)
 		{
-			CreateLeaderboardRequest createBody = new() 
+			CreateLeaderboardRequest createBody = new()
 			{
 				Name = Generators.GenerateRandomString(),
 				Slug = Generators.GenerateRandomString(),
 			};
-			HttpResponseMessage createResponse = await ApiClient.PostAsJsonAsync("/api/leaderboards", createBody, JsonSerializerOptions);
-			createResponse.EnsureSuccessStatusCode();
-			createdLeaderboards.Add(await HttpHelpers.ReadFromResponseBody<Leaderboard>(createResponse, JsonSerializerOptions));
+			createdLeaderboards.Add(
+				await HttpHelpers.Post<Leaderboard>(
+					ApiClient,
+					"/api/leaderboards",
+					new()
+					{
+						Body = createBody,
+						Jwt = Jwt,
+					}
+				)
+			);
 		}
 
 		IEnumerable<long> leaderboardIds = createdLeaderboards.Select(l => l.Id).ToList();
-		string leaderboardIdQuery = HttpHelpers.ListToQueryString(leaderboardIds, "ids");
+		string leaderboardIdQuery = ListToQueryString(leaderboardIds, "ids");
 		HttpResponseMessage getResponse = await ApiClient.GetAsync($"api/leaderboards?{leaderboardIdQuery}");
-		List<Leaderboard> leaderboards = await HttpHelpers.ReadFromResponseBody<List<Leaderboard>>(getResponse, JsonSerializerOptions);
-		foreach(var leaderboard in leaderboards)
+		List<Leaderboard> leaderboards = await HttpHelpers.ReadFromResponseBody<List<Leaderboard>>(getResponse);
+		foreach (var leaderboard in leaderboards)
```

Comment:
```suggestion
		foreach (Leaderboard leaderboard in leaderboards)
```
We're moving away from using the `var` keyword